### PR TITLE
Less restrictive v-i18n directive

### PR DESCRIPTION
**Pull request by alrusdi**

### CODE REVIEW

**Comment by nwai90** [comment]
Screenshot 2021-01-19 at 8 30 53 PM

@alrusdi There is some issue, this prints the entire json instead of just `result["message"]`

File: src/components/SelectAmount.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const SelectAmount = Vue.component('select-amount', {
   },
   template: `
     <div>
-        <div v-if="showtitle === true" v-i18n>{{ playerinput.title.toString() }}</div>
+        <div v-if="showtitle === true" v-i18n>{{ playerinput.title }}</div>
```
Comment (Lynesth) [inline src/components/SelectAmount.ts:38]: Remove the directive and use `{{ $t(playerinput.title) }}`.

Comment (alrusdi) [reply src/components/SelectAmount.ts:38]: Yeah, i see, thank you!

**Comment by alrusdi** [comment]
@nwai90 now all the problems with that mixin usage are seem to resolved

File: src/components/SelectCard.ts [hunk 2]
```diff
@@ -93,16 +95,16 @@ export const SelectCard = Vue.component('select-card', {
...
+        <div v-if="showtitle === true" class="nofloat wf-component-title">{{ $t(playerinput.title) }}</div>
         <label v-for="card in getOrderedCards()" :key="card.name" class="cardbox">
             <input v-if="playerinput.maxCardsToSelect === 1 && playerinput.minCardsToSelect === 1" type="radio" v-model="cards" :value="card" />
             <input v-else type="checkbox" v-model="cards" :value="card" :disabled="playerinput.maxCardsToSelect !== undefined && Array.isArray(cards) && cards.length >= playerinput.maxCardsToSelect && cards.indexOf(card) === -1" />
             <Card :card="card" />
         </label>
-        <div v-if="hasCardWarning()" class="card-warning" v-i18n>{{ (warning || '').toString() }}</div>
+        <div v-if="hasCardWarning()" class="card-warning">{{ $t(warning) }}</div>
         <div v-if="showsave === true" class="nofloat">
             <Button :disabled="isOptionalToManyCards() && cardsSelected() === 0" type="submit" :onClick="saveData" :title="playerinput.buttonLabel" />
```
Comment (Lynesth) [inline src/components/SelectCard.ts:106]: What about this button label ?

**Review by Lynesth — APPROVED** [review]
It looks ok now.

**Review by nwai90 — APPROVED** [review]
Tested locally, LGTM 👍

**Comment by bafolts** [comment]
I will deploy this again for today on terraforming-mars.herokuapp

File: src/components/SelectCard.ts (hunk 2 above)
Comment (alrusdi) [reply src/components/SelectCard.ts:106]: Will catch it next time

**Pull request merged**